### PR TITLE
Add support for paramaterized sql queries

### PR DESCRIPTION
**Pull request by bgschiller**

### CODE REVIEW

File: netcore-ef-util/Extensions.cs [hunk 1]
```diff
@@ -13,16 +14,27 @@ namespace SearchAThing.EFUtil
...
 
+        public static List<T> ExecSQL<T>(this DbContext context, string query, ILogger logger = null)
+        {
+            return ExecSQL<T>(context, query, logger);
+        }
+
+        public static List<T> ExecSQL<T>(this DbContext context, string query, params SqlParameter[] sqlParams)
+        {
+            return ExecSQL<T>(context, query, null, sqlParams);
+        }
```
Comment (bgschiller) [inline netcore-ef-util/Extensions.cs:25]: These additional signatures were necessary in order to allow both 
1. optional ILogger, and
2. varargs SqlParameter[]

**Comment by devel0** [comment]
thanks,
nuget package netcore-ef-util 1.0.0-CI00022 available [here](https://www.myget.org/feed/devel0/package/nuget/netcore-ef-util)

**Comment by bgschiller** [comment]
Great, thanks for merging it in so quickly!

**Comment by devel0** [comment]
added a little backward compatibility [here](https://github.com/devel0/netcore-ef-util/commit/eae94065222608186e16ecbbaf3eb94e1445f2a7)

**Comment by bgschiller** [comment]
good catch! Sorry about that.

**Pull request merged**